### ISSUE TITLE
Create ECS queries once

### ISSUE THREAD

**Issue by ddmills** [luetkemj/snail6#1]
I noticed in the systems (render and movement), the queries are being made on every `tick`, which is inefficient. The queries are meant to be created once, and it will be updated by geotic. `get()` will always return the latest cached query result
 
```js  
import ecs, { MoveTo, Position } from "../state/ecs";

const query= ecs.createQuery({
  all: [MoveTo, Position],
});

export const movement = () => {
  query.get().forEach((entity) => {
    const mPos = {
      x: entity.position.x + entity.moveTo.x,
      y: entity.position.y + entity.moveTo.y,
    };

    entity.position.x = mPos.x;
    entity.position.y = mPos.y;

    entity.remove(MoveTo);
  });
};
```

**Comment by luetkemj** [comment]
Oooh, yeah I see that now in the docs - thanks! Will update.

**Comment by luetkemj** [comment]
fixed in d0bbcb45fd3399be2cf55b5148ed138c2e5a0c34
